code/mathematical-algorithms/tower_of_hanoi/tower_of_hanoi_BinarySolution.c

```diff
@@ -0,0 +1,86 @@
+#include <stdio.h>
+
+int main()
+
+{
+  printf("Enter the number of Disks");
+  int n;
+  scanf("%d", &n);
+  for (int i = 1; i < (1 << n); i++)
+    printf("\nMove from Peg %d to Peg %d", (i & i - 1) % 3 + 1,
+           ((i | i - 1) + 1) % 3 + 1);
+  return 0;
+}
+
+/*
```

Comment:
The following comments should be compiled as the `readme`. Kindly remove from here and modify the `readme.md` file 👍